theatre/shared/src/propTypes/utils.ts

```diff
@@ -8,7 +8,9 @@ import type {
 import type {PathToProp} from '@theatre/shared/utils/addresses'
 import type {$IntentionalAny} from '@theatre/shared/utils/types'
 
-export function isPropConfigComposite(
+/** @deprecated renamed to {@link isPropConfigCompound} for clarity */
+export const isPropConfigComposite = isPropConfigCompound
+export function isPropConfigCompound(
```

Comment:
This one is undocumented, but enums would be composite but not compound.

Reply:
Okay, I will add a comment and drop this commit.

Reply:
Applied